compiler/src/scala/quoted/runtime/impl/printers/Extractors.scala

```diff
@@ -214,7 +214,7 @@ object Extractors {
       case ByNameType(underlying) =>
         this += "ByNameType(" += underlying += ")"
       case ParamRef(binder, idx) =>
-        this += "ParamRef(" += binder += ", " += idx += ")"
+        this += "ParamRef(binder, " += idx += ")"
```

Comment:
is binder meant to be a splice?

Reply:
No. If we print it we get in an infinite loop.

Reply:
See `TypeLambda` below. It was using a workaround to avoid this loop but was not printing much.

Reply:
Ah ok I realise now, you still print a placeholder because its "print as extractors"